src/main/java/fi/metatavu/famifarm/rest/V1RESTService.java

```diff
@@ -549,6 +583,20 @@ public Response deleteCampaign(UUID campaignId) {
     return createNoContent();
   }
 
+  @Override
+  @RolesAllowed({ Roles.ADMIN, Roles.MANAGER, Roles.WORKER })
+  public Response deleteCutPacking(UUID cutPackingId) {
+    fi.metatavu.famifarm.persistence.model.CutPacking existingCutPacking = cutPackingController.find(cutPackingId);
+
+    if (existingCutPacking == null) {
+      return createNotFound("Cut packing with id " + cutPackingId + " not found!");
```

Comment:
Use formatter instead of concat i.e. String.format("Product with id %s not found!", productId)